### PR TITLE
Create a sig-cli dashboard

### PR DESCRIPTION
**Pull request by fejta**
/cc @krzyzacy @mengqiy 

Expose `base_options` field
Improve config proto update instructions

### CODE REVIEW

**Comment by mengqiy** [comment]
Thanks a lot for creating this for sig-cli.
cc: @pwittrock

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1400,6 +1400,53 @@ dashboards:
...
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl slow e2e tests for master branch'
+  - name: gce-serial
+    test_group_name: ci-kubernetes-e2e-gce-serial
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl serial e2e tests for master branch'
+  - name: unit
+    test_group_name: ci-kubernetes-test-go
+    base_options: 'include-filter-by-regex=pkg/kubectl'
+    description: 'unit and integration tests for master branch'
```
Comment (mengqiy) [inline testgrid/config/config.yaml]: Here only has unit tests.
Integration tests is in `test/integration/kubectl/kubectl_test.go`.

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -1400,6 +1400,53 @@ dashboards:
...
+  - name: unit-1.6
+    test_group_name: ci-kubernetes-test-go-release-1.6
+    base_options: 'include-filter-by-regex=pkg/kubectl'
+    description: 'unit and integration tests for 1.6 branch'
```
Comment (mengqiy) [inline testgrid/config/config.yaml]: Same here.

File: testgrid/config/config.yaml [hunk 3]
```diff
@@ -1400,6 +1400,53 @@ dashboards:
...
+  - name: unit-1.5
+    test_group_name: ci-kubernetes-test-go-release-1.5
+    base_options: 'include-filter-by-regex=pkg/kubectl'
+    description: 'unit and integration tests for 1.5 branch'
```
Comment (mengqiy) [inline testgrid/config/config.yaml]: Same here.

File: testgrid/config/config.yaml [hunk 4]
```diff
@@ -1400,6 +1400,53 @@ dashboards:
...
+  - name: unit-1.4
+    test_group_name: ci-kubernetes-test-go-release-1.4
+    base_options: 'include-filter-by-regex=pkg/kubectl'
+    description: 'unit and integration tests for 1.4 branch'
```
Comment (mengqiy) [inline testgrid/config/config.yaml]: Same here.

File: testgrid/config/config.yaml (hunk 2 above)
Comment (fejta) [inline testgrid/config/config.yaml]: Ack

File: testgrid/config/config.yaml (hunk 3 above)
Comment (fejta) [inline testgrid/config/config.yaml]: Ack

File: testgrid/config/config.yaml (hunk 4 above)
Comment (fejta) [inline testgrid/config/config.yaml]: Ack

**Comment by fejta** [comment]
Agree, and showing the status of the `make test-cmd` integration tests requires addressing https://github.com/kubernetes/kubernetes/issues/45196 so that they start producing junit output

File: testgrid/config/config.yaml (hunk 1 above)
Comment (fejta) [inline testgrid/config/config.yaml]: This is not true. The test-go jobs run `make test`, `make test-cmd` and `make test-integration`. The verify jobs do not run `make test-cmd`

The test-go jobs eventually run `hack/jenkins/test-dockerized.sh` which in turn runs `make test-cmd` ([link](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh#L59))


test-go jobs https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L162
run the verify scenario without a `--script` flag, which defaults to `hack/jenkins/test-dockerized.sh` https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_verify.py

By contrast the verify jobs https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L199 
run the verify scenario with a `--script=hack/jenkins/verify-dockerized.sh`, which only runs `make verify` [link](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/verify-dockerized.sh)

Comment (mengqiy) [inline testgrid/config/config.yaml]: I see.
But I suspect filter by regex `pkg/kubectl` will only give unit tests results.

**Comment by mengqiy** [comment]
Generate junit results really should be done in the near future.
I think we can add `make test-cmd` as a single test for now if possible.

File: testgrid/config/config.yaml (hunk 1 above)
Comment (fejta) [inline testgrid/config/config.yaml]: Done


Good call! I'll add include the Overall result as well. Once these suites start producing xml results we can update the regex filter

File: testgrid/config/config.proto [hunk 5]
```diff
@@ -137,6 +137,10 @@ message DashboardTab {
   // Default code search path for changelist search links
   string code_search_path = 4;
 
+  // Base options to always include, for example:
+  // width=20&include-filter-by-regex=level_tests
```
Comment (krzyzacy) [inline testgrid/config/config.proto]: maybe include what are valid options that can be configured, like what does `include-filter-by-regex=level_tests` mean.

Comment (fejta) [inline testgrid/config/config.proto]: Done

**Comment by mengqiy** [comment]
LGTM. Thanks.
Defer the final call to @krzyzacy

**Comment by krzyzacy** [comment]
OSS rocks :-)
/lgtm

**Pull request merged**